examples/chip-tool/BUILD.gn

```diff
@@ -102,6 +102,8 @@ static_library("chip-tool-utils") {
 
   public_deps = [
     "${chip_root}/examples/common/tracing:commandline",
+    "${chip_root}/src/app/icd/client:handler",
+    "${chip_root}/src/app/icd/client:handler",
```

Comment:
```suggestion
```
Sameline was added twice?  And then the review comment pointing it out was dismissed without fixing?

Reply:
I had it removed when I accepted the review suggestion. There were merge conflicts that had to be resolved several times and it crept in again. I should have checked it.